web/src/styles/base/global.scss

```diff
@@ -347,3 +347,7 @@ ol {
 .text-no-underline-override {
   text-decoration: none;
 }
+
+.add-spacing-on-ele-scroll {
+  scroll-margin-top: 100px;
+}
```

Comment:
[dust] This might be a good time to also set `scroll-behavior: smooth`. It's not part of the scope of work, but it might be a nice thing for us to add given it's behavior that we're incorporating in other parts of the app (ex: hidden opportunities in the For You column).

https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior

Reply:
You may need to add this to a higher level container element.